src/services/handleStorage.js

```diff
@@ -1,19 +1,23 @@
 import firebase from 'firebase'
-import { db } from '../../firebase/index'
+import { db } from '../firebase/index'
 
 export async function saveToStorage(userId, key, dataSet) {
   const userDoc = db.collection(key).doc(userId)
```

Comment:
Und ich dachte gerade, was macht denn der Arzt hier? 😂  👨‍⚕️  Ist das häufig so benannt?

Reply:
haha, ja so stand es in den Dokus und bei Joe 😁